test/cqlpy/test_secondary_index.py

```diff
@@ -2100,3 +2100,12 @@ def test_limit_partition_slice(cql, test_keyspace):
         rs = cql.execute(f'SELECT pk, ck2 FROM {table} WHERE ck1 = 1 LIMIT 3')
         assert sorted(list(rs)) == [(1,1), (1,2), (2,1)]
         assert rs.has_more_pages == False
+
+def test_index_metrics(cql, test_keyspace, scylla_only):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int") as table:
+        index_name = unique_name()
+        cql.execute(f"CREATE INDEX {index_name} ON {table}(v)")
+        wait_for_index(cql, test_keyspace, index_name)
+        cql.execute(f"SELECT * FROM {table} WHERE v=1")
+        metrics = ScyllaMetrics.query(cql)
+        assert metrics.get(f'scylla_index_query_latencies_count', {"idx": index_name, "ks": test_keyspace}) == 1
```

Comment:
If you run this same test twice, it will fail because the count will be 2 instead of 1.
One option is to just check `>= 1` - if the only intention is to check that this metric "exists" and "does something".

But I think an even better approach is to check that it really counts **SELECT** and not something else. So what I would prefer to do is to get the count before running SELECT, get the count after the SELECT, and then verify that the **difference** between the two counts is exactly 1.